homework_2/task_4.py

```diff
@@ -0,0 +1,16 @@
+speed=int(input("Enter the speed rate of the car: "))
```

Comment:
ოპერატორებს შორის დასვით ცარიელი სივრცეები 
მაგალითად
```suggestion
speed = int(input("Enter the speed rate of the car: "))
```